docs/releasing.md

```diff
@@ -176,7 +161,3 @@ git new-branch rls
 git commit -am "[Lighthouse] Roll Lighthouse x.x.x\n\nBug: 772558"
```

Comment:
This commit message did not format correctly for me. The CL description imported `\n` as `\` and `n` rather than replacing with a newline character. It might be better to remove the `\n\nBug: 772558` part and add a comment saying "Set bug number to 772558"

Reply:
this is surprising to me

![image](https://user-images.githubusercontent.com/4071474/94841196-48dd8700-03df-11eb-9a57-ccb1ce763f38.png)

Reply:
It happens when `git cl upload` imports commit messages for the CL description

Reply:
oh it's because it uses the last commit message as a title, which can't have newlines.

changed to use `-b`.